feat: added launcher language option

Aireil opened the pull request:
Not sure how I feel about the LauncherLanguage.cs. I used the same idea as the client language, but there were some differences that required it.
There is probably a better way to do this; I'm open to any feedback.

Edit: I added Portuguese as I saw there was already a file for it, and the localization is complete on Crowdin. Russian and Norwegian are also 100%, but don't have a file yet.

goaaats commented on XIVLauncher/App.config:
```diff
@@ -20,6 +20,9 @@
             <setting name="Language" serializeAs="String">
                 <value>1</value>
             </setting>
+            <setting name="LauncherLanguage" serializeAs="String">
```
We aren't using these settings anymore - you don't have to add them when added to LauncherSettingsV3. They're just here for compatibility.

Aireil commented:
Removed the legacy settings and added a notice concerning the need to restart the launcher.

![XIVLauncher_2020-12-06_13-57-39](https://user-images.githubusercontent.com/33433913/101280731-095e5d00-37cb-11eb-8c0b-7e90431c27d2.png)
